Client/Ecosia/UI/MultiplyImpact/MultiplyImpact.swift

```diff
@@ -74,7 +74,7 @@ final class MultiplyImpact: UIViewController, Themeable {
     }()
     
     var referralInfo: ClimateImpactInfo {
-        .referral(value: User.shared.referrals.impact, invites: User.shared.referrals.count)
```

Comment:
I wonder if this would cause any issues with `User` decoding given users will have that stored 🤔

Reply:
It would not cause any issues as `impact` is a computed var in the Referrals model.
I did not think about removing it as part of this iteration and leaving it off the scope.
I might remove it as part of the https://github.com/ecosia/ios-core/pull/148